presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
+            throws Exception
+    {
+        try {
+            TypedSet typedSet = new TypedSet(BIGINT, 10, FUNCTION_NAME);
+
+            for (int i = 0; i <= TypedSet.FOUR_MEGABYTES + 1; i++) {
+                Block block = createLongsBlock(Collections.nCopies((int) 1, (long) i));
+
+                typedSet.add(block, 0);
+            }
+
+            fail("expected exception");
```

Comment:
In Presto, we only have empty lines for readability purpose. It's quite arbitrary and there's no rule of thumb.

Your rule sounds a little bit excessive in my opinion.

In this particular function, I would remove all empty lines. It's simple enough, and the empty lines don't add value.

I don't feel very strongly about it. Let's see what @electrum has to say.